[17.0][IMP] sale_order_type: improve readme (#3336)
OCA/sale-workflow: sale_order_type/readme/USAGE.md

1.  Go to **Sales \> Sales Orders** and create a new sale order. Select
    the new type you have created before and all settings will be
    propagated.

> ![alt text](../static/description/Sale_type_usage_1.png)

Reviewer: ```suggestion
![Sale type usage](../static/description/Sale_type_usage_1.png)
```

2.  You can also define a type for a particular partner if you go to
    *Sales & Purchases* tab of the partner and set a sale order type:

> ![alt text](../static/description/Sale_type_usage_2.png)

Reviewer: ```suggestion
![Sale type for a partner](../static/description/Sale_type_usage_2.png)
```

Scroll down to find the "Sale order Type" field:

> ![alt text](../static/description/Sale_type_usage_3.png)

Reviewer: ```suggestion
![Setting the sale type on a partner](../static/description/Sale_type_usage_3.png)
```
